### PR TITLE
MOSIP-23833

### PR DESCRIPTION
**Pull request by ckm007**

### CODE REVIEW

File: db_upgrade_scripts/mosip_kernel/sql/1.1.5.5_to_1.2.0.1-B1_upgrade.sql
Comment (sowmya695) [inline db_upgrade_scripts/mosip_kernel/sql/1.1.5.5_to_1.2.0.1-B1_upgrade.sql:1]: In this file still these tables are present in release branch.Why we are dropping the tables ?

**Review by sowmya695 — CHANGES_REQUESTED** [review]
Please check the comment

File: db_upgrade_scripts/mosip_kernel/sql/1.1.5.5_to_1.2.0.1-B1_upgrade.sql
Comment (sowmya695) [reply db_upgrade_scripts/mosip_kernel/sql/1.1.5.5_to_1.2.0.1-B1_upgrade.sql:1]: mosip_authdevice and mosip_regdevice data base changes are there in release-1.2.0.1 branch but you have created db_upgrade_scripts folder for that.whether these moved to any other repo?

File: db_upgrade_scripts/mosip_kernel/sql/1.1.5.5_to_1.2.0.1-B1_upgrade.sql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+\c mosip_kernel sysadmin
+
+DROP TABLE IF EXISTS kernel.key_alias CASCADE;
+DROP TABLE IF EXISTS kernel.key_policy_def CASCADE;
+DROP TABLE IF EXISTS kernel.key_policy_def_h CASCADE;
+DROP TABLE IF EXISTS kernel.key_store CASCADE;
+DROP TABLE IF EXISTS kernel.sync_control CASCADE;
+DROP TABLE IF EXISTS kernel.sync_job_def CASCADE;
+DROP TABLE IF EXISTS kernel.sync_transaction CASCADE;
+DROP TABLE IF EXISTS kernel.dao_key_store CASCADE;
```
Comment (vishwa-vyom) [inline db_upgrade_scripts/mosip_kernel/sql/1.1.5.5_to_1.2.0.1-B1_upgrade.sql:10]: As per 1.1.5.5 I don't see these tables in mosip_kernel, so we have to remove this section completely

File: db_upgrade_scripts/mosip_kernel/sql/1.1.5.5_to_1.2.0.1-B1_upgrade.sql
Comment (vishwa-vyom) [reply db_upgrade_scripts/mosip_kernel/sql/1.1.5.5_to_1.2.0.1-B1_upgrade.sql:1]: In LTS, I think both regdevice and authdevice has be moved inside pms, so these dbs are no more required.

**Review by vishwa-vyom — APPROVED** [review]

**Pull request merged**